var/spack/repos/builtin/packages/fio/package.py

```diff
@@ -14,8 +14,9 @@ class Fio(AutotoolsPackage):
     """
 
     homepage = "https://github.com/axboe/fio"
-    url = "https://github.com/axboe/fio/archive/fio-3.25.tar.gz"
+    url = "https://github.com/axboe/fio/archive/fio-3.26.tar.gz"
```

Comment:
Just an FYI: You do not have to change this for new versions, Spack will determine it automatically.